extensions/azuremonitor/package.json

```diff
@@ -178,20 +178,20 @@
     },
     "languages": [
       {
-        "id": "loganalytics",
+        "id": "azuremonitorlogs",
         "aliases": [
-          "LogAnalytics",
-          "loganalytics"
+          "AzureMonitorLogs",
+          "azuremonitorlogs"
         ],
         "extensions": [
-          ".loganalytics"
+          ".azuremonitorlogs"
```

Comment:
Is the loganalytics extension deprecated then? Or should you be able to support both?